java/app/src/main/res/layout/fragment_checkout.xml

```diff
@@ -8,17 +8,34 @@
             type="com.fullstorydev.shoppedemo.ui.checkout.CheckoutViewModel" />
         <variable
             name="handlers"
-            type="com.fullstorydev.shoppedemo.ui.checkout.CheckoutEventHandlers"/>
+            type="com.fullstorydev.shoppedemo.ui.checkout.CheckoutEventHandlers" />
     </data>
 
     <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:fillViewport="true">
```

Comment:
everything below are just changing from native components to MDC and adding styling to the xml. +a handler change from `Spinner` to `AutoCompleteTextView`